lib/sycamore/sycamore/docset.py

```diff
@@ -974,29 +980,51 @@ def llm_filter(
             new_field: The field that will be added to the DocSet with the outputs.
             prompt: LLM prompt.
             field: Document field to filter based on.
-            threshold: Cutoff that determines whether or not to keep document.
+            threshold: Cutoff that determines whether to keep document.
```

Comment:
Can you make this more clear? Maybe: "If the value of the computed result is an integer value greater than or equal to this threshold, the document will be kept."

Reply:
Ack